packages/core/src/Cardano/types/AuxiliaryData.ts

```diff
@@ -26,6 +26,6 @@ export interface AuxiliaryDataBody {
 }
 
 export interface AuxiliaryData {
-  hash?: Cardano.Hash16;
+  hash?: Cardano.Hash16; // TODO: Review: need to find an example of this to verify type and length
```

Comment:
Transaction and block hashes are also `Hash16` type, which can be validated as 64 character strings

https://explorer.cardano.org/en/transaction?id=61f85fad0dbabcafa9cf1d3